pwa/static/themes/utrecht.css

```diff
@@ -157,14 +157,23 @@
   /* @conduction: Secondary Top Navigation */
   --web-app-secondary-top-nav-item-padding: var(--web-app-size-md);
   --web-app-secondary-top-nav-font-size: var(--web-app-font-size-md);
-  --web-app-secondary-top-nav-color: var(--web-app-color-black);
+  --web-app-secondary-top-nav-color: var(--web-app-color-black) !important;
   --web-app-secondary-top-nav-background-color-hover: rgba(0, 0, 0, 0.1);
   --web-app-secondary-top-nav-font-weight: var(--web-app-font-weight-normal);
 
   /* @condcution: Primary Top Navigation */
   --web-app-primary-top-nav-item-padding: var(--web-app-size-md);
-  --web-app-primary-top-nav-color: var(--web-app-color-white);
+  --web-app-primary-top-nav-color: var(--web-app-color-black) !important;
   --web-app-primary-top-nav-background-color: hsl(211 60% 35%);
-  --web-app-primary-top-nav-background-color-hover: rgba(255, 255, 255, 0.2);
+  --web-app-primary-top-nav-background-color-hover: rgba(255, 255, 255, 0.2) !important;
```

Comment:
We're setting `!important` here and not in the other theme(s), do we need it?

If so, is this something we could fix within our implementation, so that its not needed anymore?